config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
No, I think adding the dummy app is okay. Actually we don't even need to do that since we don't release the gem on rubygems. With git references, the whole repo is present anyway.